### ISSUE TITLE
pylint reports E0401:import error for the mido library

### ISSUE THREAD

**Issue by ReddyKilowatt** [microsoft/vscode-pylint#556]
[pylint_bug.txt](https://github.com/microsoft/vscode-pylint/files/15367394/pylint_bug.txt)

**Comment by ReddyKilowatt** [comment]
Why does this website ask for trace files that are too long to paste into bug reports???????????????????????????

**Comment by ReddyKilowatt** [comment]
It looks like my venv was the problem.
Somehow, the packages I installed into the venv (yes, it was activated) were installed in the global python env.

**Comment by karthiknadig** [comment]
> Somehow, the packages I installed into the venv (yes, it was activated) were installed in the global python env.

This can happen if you run ‘pip install’ vs `python -m pip install`. See here for more details: https://snarky.ca/why-you-should-use-python-m-pip/

**Comment by ReddyKilowatt** [comment]
@karthiknadig I know about python -m pip install xxx
After I discovered the problem, I had tried adding packages to the venv, but they still went to the global python.
I have never had this problem with Python on Windows until now. I ended up deleting the venv, and Python, and reinstalling Python for all users, so that it doesn't get installed in my user directory (Appdata ....)
Now, the venv seems to work much better, and vscode is able to resolve most python symbols/objects now.
